tests/testthat/helper-outpack-http.R

```diff
@@ -2,7 +2,7 @@ mock_headers <- function(...) {
   structure(list(...), class = c("insensitive", "list"))
 }
 
-mock_response <- function(content, status = 200, wrap = TRUE, download = NULL) {
+mock_response <- function(content, status = 200L, wrap = TRUE, download = NULL) {
```

Comment:
```suggestion
mock_response <- function(content, status = 200L, wrap = TRUE, 
                          download = NULL) {
```

this will keep the linter happy